### PR TITLE
v2

### PR DESCRIPTION
**Pull request by BitcoinOG**
Preview: https://opendex.network/v/develop

DONE:
Refactored XU->OpenDEX
Renamed "market maker" -> "liquidity provider"
WebApp as "swap", e.g. Boltz is a "Swap Provider", all things on the OpenDEX Network is called "trading".
Integrated Docs with existing site & BOLDs

Ignore some linking issues that brings up the md on github instead of the gitbook page, I expect that to fix itself once merged into master.

**After merge:** change 4 header items to `Swap WebApp | Desktop App | Support | Twitter`

TODO:
- https://github.com/opendexnetwork/opendex/issues/48
- https://github.com/opendexnetwork/opendex/issues/49
- https://github.com/opendexnetwork/opendex/issues/50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,32 @@
...
+**OpenDEX is a cross-chain DEX ecosystem, featuring a user-friendly webapp for traders and a high-speed network built on the [Lightning](https://lightning.network/) and [Connext](https://connext.network/) networks for liquidity providers.** OpenDEX consists of software, community, member projects, and an open protocol standard with the goal to unify currently incompatible protocols and fragmented liquidity.
 
 ## Get Started
 
--&gt; [Read the BOLD specifications](read/00-introduction.md)
+-> [**Trade** securely via an easy-to-use webapp](https://boltz.exchange/)
```
Comment (maybeast) [inline README.md]: Incorrect link?

File: README.md [hunk 2]
```diff
@@ -2,33 +2,32 @@
...
 
--&gt; [Contribute to the BOLD specifications](community/contribute.md)
+-> [**Provide liquidity** running the market maker setup](https://boltz.exchange/)
```
Comment (maybeast) [inline README.md]: Incorrect link?

File: README.md [hunk 3]
```diff
@@ -2,33 +2,32 @@
...
+* [Tor](https://www.torproject.org/) by default to preserve privacy.
+* Open, transparent protocol. Support for multiple implementations.
 
 ## Support & Community
+**OpenDEX is not a company**, nor any other sort of legal entity. It is an open community of individuals & member projects, working together to maintain the OpenDEX software and protocol specifications. OpenDEX was started to create a long-term viable alternative to trading on trusted and KYCed centralized exchanges. Find the slides of the first public announcement at \#hcpp19 in Prague [here](https://github.com/opendexnetwork/opendex/tree/90dd7a7e28cebf3d9cafc3f0a2218362552c06d6/slides/20191005_hcpp19.pdf) and the video recording [here](https://www.youtube.com/watch?v=euSr9A6tI90).
 
-* [Contribute](community/contribute.md)
-* Chat with us on [Discord](https://discord.gg/RnXFHpn)
-
+* Join our [weekly community call or watch the recordings](community/videos#community-calls).
```
Comment (maybeast) [inline README.md]: In preview mode this gives me a 404 error.

File: docs/BeastXUD.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+   * A NAS/Samba share works too.
+
+## Setup
+
+1. [Download Ubuntu Server 20.04 LTS](https://ubuntu.com/download/server) onto your computer. Any other linux distribution supporting docker is also fine. This guide was written assuming `Ubuntu Server 20.04 LTS`.
+2. Insert a USB Stick into your computer and [create the a bootable USB Stick](https://ubuntu.com/tutorials/tutorial-create-a-usb-stick-on-ubuntu) with the ubuntu image you just downloaded.
+3. Open your Mini PC, plug in RAM & drives, close it, connect it to your router via ethernet cable and to a power supply. Connect a screen via HDMI, a USB keyboard, the created bootable USB Stick, fire the Mini PC up and follow the the inital setup instructions.
+4. Update ubuntu via `sudo apt update && sudo apt upgrade`
+5. If you are using Ubuntu 20.04 LTS, install docker by running `sudo apt install docker.io`. Otherwise if you are using any version besides Ubuntu 20.04, follow the [official instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/) to install docker.
+6. Add new user `xud`:
```
Comment (maybeast) [inline docs/BeastXUD.md]: Change username to `opendex`?

**Review by maybeast — COMMENTED** [review]
There are many references of xud still in the docs. Do we want to clean them in this PR?

File: README.md (hunk 1 above)
Comment (BitcoinOG) [reply README.md]: Nope, correct (for now).

File: README.md (hunk 2 above)
Comment (BitcoinOG) [reply README.md]: Yup, fixed.

**Comment by BitcoinOG** [comment]
This is *not* ready for end-to-end review yet @maybeast , will take it out of draft once it is

**Comment by BitcoinOG** [comment]
This is ready for in-depth review now. And yes I'll merge this as one clean commit ;)

File: docs/Contribute.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+### Commenting & Documentation
+
+Make your code legible by using descriptive variable, function, and class names. For blocks of codes whose function is not readily apparent, add comments explaining what they do. If a pull request changes the interface to `opendexd` or introduces new functionality, update the README to describe these changes.
+
+`opendexd` uses [TypeDoc](http://typedoc.org/guides/doccomments/) and any comments documenting specific classes, methods, properties should follow this convention.
+
+Commit messages should be concise and descriptive. For larger or more complex commits, add details of what has changed in the commit description.
+
+If you see something missing or want to develop another cool feature or bug fix which doesn't have an issue yet, open one and we'll review and give it a price tag if you ask nicely ;) ! We don't recommend working on things without a GitHub issue.
```
Comment (stackingcats) [inline docs/Contribute.md]: Is the price tag part relevant atm?

File: docs/Dockerless Guide.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+disable = true
+```
+
+For convenience, consider adding `alias opendex-cli='/home/opendexd/opendexd/bin/opendex-cli -p 8886'` to the opendexd user's `.bashrc` and source it. Then restart `opendexd` once (e.g. with `systemctl restart opendexd`) and try running `opendex-cli getinfo`, which should return with an overview of opendexd's, as well as lnd's and connext status.
+
+Ping us in the help channel of our [Discord server](https://discord.gg/RnXFHpn) for support.
+
+# Tips 'n Tricks
+
+* When installing on a Raspberry Pi you might see `Unexpected error during initialization`. [Here](https://github.com/ExchangeUnion/xud/issues/1199#issuecomment-527819108) the solution.
```
Comment (stackingcats) [inline docs/Dockerless Guide.md]: Here _is_ the solution?

File: docs/Pro Hardware Guide.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
+This guide is written for professional liquidity providers build an always-on OpenDEX node with a powerful Mini PC.
```
Comment (stackingcats) [inline docs/Pro Hardware Guide.md]: There's probably something missing in this sentence.

File: docs/Pro Hardware Guide.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+```
+8. Looking good! Optionally, add an alias to enter your opendexd environment by simply typing "opendex":
+```bash
+opendex@ubuntu:~$ sudo nano ~/.bash_aliases
+# add the line
+alias opendex='bash ~/opendex.sh'
+# CTRL+S, CTRL+X. Then run
+opendex@ubuntu:~$ source ~/.bashrc
+```
+9. Connect the USB stick to your machine and set it up. It is very important to do this for a mainnet setup (given you do not want to loose money)!
```
Comment (stackingcats) [inline docs/Pro Hardware Guide.md]: loose -> lose

**Review by stackingcats — COMMENTED** [review]
Just some minor nits discovered.

**Comment by BitcoinOG** [comment]
> Just some minor nits discovered.

    Please list em :)

**Comment by BitcoinOG** [comment]
> > Just some minor nits discovered.
> 
>     Please list em :)

Ah sry, the GH mobile app didnt display your comments. Thx!

File: docs/Contribute.md (hunk 5 above)
Comment (BitcoinOG) [reply docs/Contribute.md]: Nah better not, fixed!

File: docs/Dockerless Guide.md (hunk 6 above)
Comment (BitcoinOG) [reply docs/Dockerless Guide.md]: Fixed

File: docs/Pro Hardware Guide.md (hunk 7 above)
Comment (BitcoinOG) [reply docs/Pro Hardware Guide.md]: Fixed

File: docs/Pro Hardware Guide.md (hunk 8 above)
Comment (BitcoinOG) [reply docs/Pro Hardware Guide.md]: Fixed

File: docs/Liquidity Provider Guide.md [hunk 9]
```diff
@@ -0,0 +1,393 @@
...
+# The Setup
+
+From here we assume that your device is running with docker set up. Check the guides in the hardware section above if your device is not ready yet.
+
+## Let's Roll
+
+Start the environment with
+
+```bash
+curl https://raw.githubusercontent.com/opendexnetwork/opendex-docker/master/opendexd.sh -o ~/opendexd.sh
```
Comment (maybeast) [inline docs/Liquidity Provider Guide.md:51]: Do we want to upgrade these to use the new launcher binary instead?

File: docs/Overview.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+## How to Run a Node
+
+👉 as [**Liquidity Provider**](Liquidity%20Provider%20Guide.md), earning via automated arbitrage between external exchanges and OpenDEX
+
+👉 as [**Swap Provider**](Swap%20Provider%20Guide.md), sourcing liquidity on OpenDEX **(WIP)**
+
+👉 as [**Day Trader**](Day%20Trader%20Guide.md), trading instantaneously while preserving full control & privacy
+
+👉 as [**Developer**](Developer%20Guide.md), contributing or building on top of `opendexd`
```
Comment (maybeast) [inline docs/Overview.md:13]: The current developer-guide.md is quite out of date. Maybe just point them to https://github.com/opendexnetwork/opendexd#installation

File: docs/Liquidity Provider Guide.md (hunk 9 above)
Comment (maybeast) [reply docs/Liquidity Provider Guide.md:51]: Or hmm... launcher currently does not have interactive mode if I'm not mistaken. :thinking:

**Review by maybeast — COMMENTED** [review]
Looks good in general. Left some questions regarding `opendexd.sh`.

File: docs/Liquidity Provider Guide.md (hunk 9 above)
Comment (BitcoinOG) [reply docs/Liquidity Provider Guide.md:51]: > Do we want to upgrade these to use the new launcher binary instead?

Yes.

> Or hmm... launcher currently does not have interactive mode if I'm not mistaken. thinking

We'll need this rather sooner than later. I quite some situations I can't access any webapp, e.g. setting up opendex-docker on a VPS. Can we bring back interactive mode for create flow and usage via CLI? @Maith8Ae

File: docs/Overview.md (hunk 10 above)
Comment (BitcoinOG) [reply docs/Overview.md:13]: What exactly is out of date? The main purpose of this guide is connecting a native opendexd to dependencies within opendex-docker.

File: docs/Liquidity Provider Guide.md (hunk 9 above)
Comment (maybeast) [reply docs/Liquidity Provider Guide.md:51]: I'd expect this to not be super quick effort. Are we still `opendexd.sh` compatible for interactive mode @Maith8Ae?

If so, perhaps we can merge as is and do a follow-up PR once the launcher interactive mode is ready. This PR is huge and quite difficult to review as it covers a lot.

File: docs/Overview.md (hunk 10 above)
Comment (maybeast) [reply docs/Overview.md:13]: Actually, in hindsight. It should work, but I haven't tested it. 

My point was that maybe we can keep everything "development" docs in the opendexd repo install guide, but I now realize that it's 2 different guides:
- opendexd repo guide -> setup everything manually
- developer guide in docs -> use external opendexd with opendex-docker

File: docs/Liquidity Provider Guide.md (hunk 9 above)
Comment (BitcoinOG) [reply docs/Liquidity Provider Guide.md:51]: Agreed, add a WIP disclaimer which we can remove once interactive CLI mode is back: https://opendex.network/v/develop/docs/liquidity-provider-guide#lets-roll-wip-not-fully-working-yet

**Review by maybeast — APPROVED** [review]

File: docs/Overview.md (hunk 10 above)
Comment (BitcoinOG) [reply docs/Overview.md:13]: Correct

**Pull request merged**